src/applications/check-in/pre-check-in/actions/index.js

```diff
@@ -27,3 +29,21 @@ export const setVeteranData = ({ appointments, demographics }) => {
     payload: { appointments, demographics },
   };
 };
+
+export const UPDATE_FORM = 'UPDATE_FORM';
+
+export const updateFormAction = ({
```

Comment:
I like this, this is what I was envisioning for how to update forms. Good Work 🦖 🌮